tools/misc/codeowners.json

```diff
@@ -0,0 +1,73 @@
+{
+    "git_account" : "ARMmbed",
+    "repository" : "mbed-os",
+    "default" : "mbed-os-maintainers",
+    "owner_list" : {    
+        "platform" : ["SenRamakri", "c1728p9", "geky"],
+
+        "TESTS/mbed_functional" : ["SenRamakri", "c1728p9", "geky"],
+
+        "events" : ["mbed-os-maintainers"],
+        "TESTS/events" : ["mbed-os-maintainers"],
+
+        "drivers" : ["bulislaw", "c1728p9", "pan-", "scartmell-arm", "SenRamakri"],
+        "TESTS/mbed_drivers" : ["bulislaw", "c1728p9", "pan-", "scartmell-arm", "SenRamakri"],
+
+        "tools" : ["theotherjimmy"],
+
+        "rtos" : ["bulislaw", "c1728p9", "pan-", "scartmell-arm"],
+        "TESTS/mbedmicro-rtos-mbed" : ["bulislaw", "c1728p9", "pan-", "scartmell-arm"],
+
+        "hal" : ["bulislaw", "c1728p9", "pan-", "scartmell-arm"],
+        "TESTS/mbed_hal" : ["bulislaw", "c1728p9", "pan-", "scartmell-arm"],
+
+        "features/FEATURE_BLE" : ["pan-"],
```

Comment:
Could you add @paul-szczepanek-arm in the list of reviewers ? 
"features/FEATURE_BLE/targets/TARGET_NORDIC" should also *summon* @ARMmbed/team-nordic

Reply:
+1